core/src/fight/backgrounds.ts

```diff
@@ -1,30 +1,36 @@
+export enum MapEffect {
+	HalfSpeed,
+  OneHp,
```

Comment:
I don't like this modifier, fighters having only one HP doesn't make good fights.

Reply:
OK, I will remove it.